src/main/java/uk/gov/ons/ctp/response/action/representation/ActionRulePutRequestDTO.java

```diff
@@ -0,0 +1,31 @@
+package uk.gov.ons.ctp.response.action.representation;
+
+import lombok.AccessLevel;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import javax.validation.constraints.Max;
+import javax.validation.constraints.Min;
+import javax.validation.constraints.Size;
+
+/**
+ * Domain model object for representation.
+ */
+@Data
+@AllArgsConstructor
+@NoArgsConstructor(access = AccessLevel.PUBLIC)
```

Comment:
Shouldn't this only have an AllArgsConstructor as all are required for a valid PUT?

Reply:
Is this because priority isn't required?

Reply:
No they are all optional, the implementation only updates the fields that are provided.